src/test/java/com/jcabi/beanstalk/maven/plugin/GenericMojoTest.java

```diff
@@ -147,4 +151,78 @@ public void validatesGoodYaml() throws Exception {
             )
         );
     }
+
+    /**
+     * Verifies that execute throws an exception, if there is no .ebextensions
+     * directory in the WAR file.
+     * @throws Exception Thrown in case of error.
+     */
+    @Test
+    public void checkEbextensionsValidityThrowsExceptionNoDir()
+        throws Exception {
+        // @checkstyle IllegalTypeCheck (2 lines)
+        final AbstractBeanstalkMojo mojo =
+            Mockito.mock(AbstractBeanstalkMojo.class);
+        final File mockFile = Mockito.mock(File.class);
+        final ZipFile mockZipFile = Mockito.mock(ZipFile.class);
+        Mockito.when(mockFile.exists()).thenReturn(true);
+        Mockito.when(mojo.createZipFile()).thenReturn(mockZipFile);
+        Mockito.doCallRealMethod().when(mojo).execute();
+        Mockito.doCallRealMethod().when(mojo)
+            .setWar(Mockito.any(File.class));
+        Mockito.doCallRealMethod().when(mojo)
+            .validate(Mockito.any(ZipFile.class));
+        mojo.setWar(mockFile);
+        Mockito.when(mockZipFile.getEntry(".ebextensions")).thenReturn(null);
+        try {
+            mojo.execute();
+            Assert.fail("Expect MojoFailureException to be thrown");
+        } catch (final MojoFailureException exception) {
+            MatcherAssert.assertThat(
+                exception.getMessage(),
+                Matchers.equalTo(
+                    ".ebextensions directory does not exist in the WAR file"
+                )
+            );
+        }
+    }
+
+    /**
+     * Verifies that execute throws an exception, if the .ebextensions directory
+     * is empty.
+     * @throws Exception Thrown in case of error.
+     */
+    @Test
+    @SuppressWarnings("unchecked")
+    public void checkEbextensionsValidityThrowsExceptionNoConfigFiles()
```

Comment:
@westonized the convention is described on http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names, but it is not yet enforced with Qulice. It will be at some point